Add views support in iceberg connector (#19872)
prestodb/presto: presto-docs/src/main/sphinx/connector/iceberg.rst

    ``Day``, ``Month``, ``Year``, ``Hour`` partition column transform functions are not supported in Presto Iceberg
    connector yet (:issue:`20570`).

CREATE VIEW
^^^^^^^^^^^^

> Currently Iceberg connector supports creating views using hive and glue metastores.

Reviewer: ```suggestion
The Iceberg connector supports creating views of Hive and Glue metastores.
```

Reviewer: @agrawalreetika Yes! I like your suggestion better.

    ``Day``, ``Month``, ``Year``, ``Hour`` partition column transform functions are not supported in Presto Iceberg
    connector yet (:issue:`20570`).

CREATE VIEW
^^^^^^^^^^^^

> Currently Iceberg connector supports creating views in Hive and Glue metastores.

Author: Changed.

> Create a view named ``view_page_views`` on top of ``iceberg.web.page_views`` table created above as given below::

Reviewer: ```suggestion
To create a view named ``view_page_views`` for  the ``iceberg.web.page_views`` table created in the `CREATE TABLE`_ example::
```

Because documentation pieces can get moved around, "above" and "below" are fragile. Avoiding them when possible can future-proof the documentation from becoming out of date through rearranging. Added a link to the CREATE TABLE example to help the user find it - this is a long page and they could scroll past it looking for it.

Author: Changed.
